### DISCUSSION TITLE
Is there a way to hide the message "No matches for 'wildcard', used '**/wildcard'"?

### DISCUSSION THREAD

**Discussion by Alectriciti** [adieyal/sd-dynamic-prompts#582] [General]
Say I have a wildcard file:

**/wildcards/colors/red.txt**

I want to access it like this:
> \_\_red\_\_

not like this:
>\_\_color/red\_\_
>\_\_\*\*/red\_\_

It technically works, but the real issue is getting spammed with this message:

> "No matches for wildcard 'color', used '\*\*\/color' to match"

Don't get me wrong, globbing is very useful in a some situations, But the main reason I do it like this for a lot of my wildcards is for a tidier prompt and readability. Dynamic Prompting can already get pretty visually chaotic as it is.

The nice thing is, it technically works as intended. But sometimes getting spammed with that message in the console makes me want to revert to my older build. This is particularly helpful when trying to work out actual issues like typos.

Didn't feel it was appropriate as a suggestion but rather a discussion. It's possible I'm just not doing something as intended. But at the end of the day, this is just a quality of life thing and doesn't really stop me from having it work how I need it to. Thanks for reading, and thanks for this awesome extension. Seriously, it's the best.

**Comment by adieyal** [comment]
Try adding this to webui-user.sh

`export SD_WEBUI_LOG_LEVEL=error`

or if you're on Windows, then in webui-user.bat
`set SD_WEBUI_LOG_LEVEL=error`

It's not the best solution since it will silence all warnings. If you are happy to edit some python code, then you could probably add the following to webui.py. Somewhere near the top where the other logging statements are, add this:

`logging.getLogger("dynamicprompts.wildcards.wildcard_manager").setLevel(logging.ERROR)`
